src/main.rs

```diff
@@ -65,18 +69,38 @@ fn main() {
 
     ::std::process::exit(match args.subcommand() {
         ("init", _) => FuzzProject::init().map(|_| ()),
-        ("add", matches) =>
-            FuzzProject::new().and_then(|p| p.add_target(matches.expect("arguments present"))),
-        ("list", _) => FuzzProject::new().and_then(|p| p.list_targets()),
-        ("run", matches) =>
-            FuzzProject::new().and_then(|p| p.exec_target(matches.expect("arguments present"))),
+        ("add", matches) => FuzzProject::new()
+            .and_then(|p| p.add_target(matches.expect("arguments present"))),
+        ("list", _) => FuzzProject::new()
+            .and_then(|p| p.list_targets()),
+        ("run", matches) => FuzzProject::new()
+            .and_then(|p| p.exec_fuzz(matches.expect("arguments present"))),
+        ("cmin", matches) => FuzzProject::new()
+            .and_then(|p| p.exec_cmin(matches.expect("arguments present"))),
+        ("tmin", matches) => FuzzProject::new()
+            .and_then(|p| p.exec_tmin(matches.expect("arguments present"))),
         (s, _) => panic!("unimplemented subcommand {}!", s),
     }.map(|_| 0).unwrap_or_else(|err| {
         utils::report_error(&err);
         1
     }));
 }
 
+fn fuzz_wrapper(name: &str) -> App {
+    SubCommand::with_name(name)
+        .arg(Arg::with_name("no_opt").long("no-opt").short("no")
```

Comment:
So of this change was to switch what happens when you just run `cargo fuzz run TARGET` -- right now, this fuzzes an unoptimized binary with debug assertions whereas IMO we should default to fuzzing an optimized binary with debug assertions, and I'm not sure how else to do that without changing the CLI.

This also gives the user strictly more flexibility -- right now, there's no way to build an unoptimized binary without debug assertions (although I'm not sure why you'd want to 😜), whereas this change lets you do all 4 combinations of unoptimized/optimized and debug asserts / no debug asserts.